update/errors.go

```diff
@@ -197,5 +194,5 @@ var ErrNilPendingTxProcessor = errors.New("nil pending tx processor")
 // ErrNilHardForkBlockProcessor signals that nil hard fork block processor has been provided
 var ErrNilHardForkBlockProcessor = errors.New("nil hard fork block processor")
 
-// ErrNilTrieFactory signals that nil trie factory has been provided
-var ErrNilTrieFactory = errors.New("nil trie factory")
+// ErrNilTrieStorageManager signals that nil trie storage manager has been provided
+var ErrNilTrieStorageManager = errors.New("nil storage manager has been factory")
```

Comment:
Refactor this error message: -> "nil trie storage manager"

Reply:
done